lib/galaxy/config/schema.py

```diff
@@ -33,8 +37,68 @@ def __init__(self, schema_path, app_name):
         self.raw_schema = self._read_schema(schema_path)
         self.description = self.raw_schema.get("desc", None)
         app_schema = self.raw_schema['mapping'][app_name]['mapping']
-        super(AppSchema, self).__init__(app_schema)
+        self._preprocess(app_schema)
+        super().__init__(app_schema)
```

Comment:
This syntax is Python3-only, I don't think we are at that stage yet.

Reply:
I wasn't sure how Python3-only our code could be. I'll keep it Python2-compatible.